src/app/Core/api.service.ts

```diff
@@ -0,0 +1,20 @@
+export interface ApiResponse<T>{
+    apiStatus: number;
+    apiProp?: T;

```

Comment:
so you are in the api interface i would use just status and data as names.

Reply:
Done.